apps/charterafrica/src/pages/index.page.js

```diff
@@ -42,6 +42,47 @@ export async function getStaticProps({ defaultLocale, locale, locales }) {
     fallbackLocale: defaultLocale,
   });
 
+  const { docs: homePage } = await payload.findPage("home", {
+    locale,
+    fallbackLocale: defaultLocale,
+  });
+
+  const { blocks: pageBlocks } = homePage[0] ?? {
+    blocks: [],
+  };
+
+  const blocks =
+    pageBlocks?.map(({ blockType, ...other }) => ({
+      ...other,
+      slug: blockType,
+    })) || [];
+
```

Comment:
Use changes [suggested in the ecosystem PR](https://github.com/CodeForAfrica/ui/pull/291#discussion_r1072239808).